QgisModelBaker/gui/workflow_wizard/project_creation_page.py

```diff
@@ -63,6 +62,16 @@ def __init__(self, parent, title):
 
         self.create_project_button.clicked.connect(self._create_project)
 
+        self.is_complete = False
+
+    def isComplete(self):
```

Comment:
Random suggestion (because I had to lookup the `QWizardPage` documentation to find that this overrides a virtual method and this would have helped me see this directly from here): https://stackoverflow.com/a/8313042/2319028

Reply:
I like it. 

Is this something we do as best-practice?